[2018-11-28T12:23:15.035] [ERROR] Error - {"Error_Name":"Mongodb Connection Error","Error_Address":"Server.js","Error_Message":{"name":"MongoNetworkError","message":"failed to connect to server [ds249311.mlab.com:49311] on first connect [MongoNetworkError: getaddrinfo EAI_AGAIN ds249311.mlab.com:49311]"}}
[2018-11-28T12:23:15.040] [ERROR] Error - {"Error_Name":"","Error_Address":"","Error_Message":{"name":"MongoNetworkError","message":"failed to connect to server [ds249311.mlab.com:49311] on first connect [MongoNetworkError: getaddrinfo EAI_AGAIN ds249311.mlab.com:49311]"}}
[2018-11-28T12:55:48.474] [ERROR] Error - {"Error_Name":"Mongodb Connection Error","Error_Address":"Server.js","Error_Message":{"name":"MongoNetworkError","message":"failed to connect to server [ds249311.mlab.com:49311] on first connect [MongoNetworkError: getaddrinfo EAI_AGAIN ds249311.mlab.com:49311]"}}
[2018-11-28T12:55:48.479] [ERROR] Error - {"Error_Name":"","Error_Address":"","Error_Message":{"name":"MongoNetworkError","message":"failed to connect to server [ds249311.mlab.com:49311] on first connect [MongoNetworkError: getaddrinfo EAI_AGAIN ds249311.mlab.com:49311]"}}
[2018-11-28T14:52:39.663] [ERROR] Error - {"Error_Name":"Mongodb Connection Error","Error_Address":"Server.js","Error_Message":{"name":"MongoNetworkError","message":"failed to connect to server [ds249311.mlab.com:49311] on first connect [MongoNetworkError: getaddrinfo EAI_AGAIN ds249311.mlab.com:49311]"}}
[2018-11-28T14:52:39.668] [ERROR] Error - {"Error_Name":"","Error_Address":"","Error_Message":{"name":"MongoNetworkError","message":"failed to connect to server [ds249311.mlab.com:49311] on first connect [MongoNetworkError: getaddrinfo EAI_AGAIN ds249311.mlab.com:49311]"}}
